<!-- This is just a quick pipeline hack to get training data for the GRSD feature -->

<launch>

  <group ns="object_classification_training">

    <!--From here on we will work in /map coordinate frame -->
    <param name="global_frame_id" value="/map" />	

    <!-- Table Object Detector (passive version) -->
    <node pkg="cloud_tools" type="table_object_detector_passive_node_one_table" name="table_object_detector_passive" output="screen"  respawn="true">
      <param name="input_cloud_topic" value="/shoulder_cloud" />
      <param name="table_delta_z" value="0.02" />
      <param name="output_table_topic" value="/object_classification_training/one_table_with_objects" />
      <param name="sac_distance_threshold" value="0.015" />
      <param name="normal_eps_angle" value="5" />
      <param name="table_delta_z" value="0.01" />
    </node>
  
    <!-- Table Message Splitter -->
     <node pkg="cloud_tools" type="table_msg_split_node" name="table_msg_split_node" output="screen">
       <param name="input_table_topic" value="/object_classification_training/one_table_with_objects" />
       <param name="output_pcds_topic" value="/object_classification_training/table_objects" />
       <param name="output_polygon_topic" value="/object_classification_training/table_polygon" />
     </node>

  </group>

</launch>

